### PR TITLE
9.11.0 RN

### PR DESCRIPTION
**Pull request by Adam-Dupaski**

### CODE REVIEW

File: content/releasenotes/studio-pro/9.11.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+* We fixed an issue where the passwords of demo users did not update when changing the password policy.
+* We fixed a bug that caused an error when trying to publish an entity as an OData resource while including system members `owner` or `changedBy` as attributes. We also made sure these system attributes are listed in `NonUpdatable(Navigation)Properties` in the service metadata XML.
+* We fixed an issue in updatable published OData resources for exposed system members. Now, these members are properly listed in the `NonUpdatable(Navigation)Properties` in the metadata, and the runtime gives a clear error message when a client attempts to update them.
+* We fixed an issue in the [Call REST service](/refguide/call-rest-action) activity where the data entered for a custom request template disappeared.
+* We fixed an issue where a pluggable widget data source having XPath-based constraints where a member did not have read access resulted in an error.
+* We fixed an issue with the [tab container](/refguide/tab-container) where its content was not updating upon switching tabs with visibility constraints.
+
```
Comment (KevinVlaanderen) [inline content/releasenotes/studio-pro/9.11.md:65]: * We fixed an issue regarding the sort order with more than four attributes in XPath queries. (Ticket 141117)

Comment (KevinVlaanderen) [reply content/releasenotes/studio-pro/9.11.md:65]: * We fixed sorting by multiple attributes in Excel export of Data grid. (Ticket 136239)

**Review by KevinVlaanderen — DISMISSED** [review]

File: content/releasenotes/studio-pro/9.11.md (hunk 1 above)
Comment (Adam-Dupaski) [reply content/releasenotes/studio-pro/9.11.md:65]: https://github.com/mendix/docs/pull/4191/commits/f5133dbc69942e9a082e3830241cf1fa88d54356

Comment (Adam-Dupaski) [reply content/releasenotes/studio-pro/9.11.md:65]: https://github.com/mendix/docs/pull/4191/commits/861c267a3f7a26e6fbc2c5a413d0743832ee5ceb

File: content/releasenotes/studio-pro/9.11.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+* We added support to autocomplete XPath and OQL with the <kbd>Tab</kbd> key.
+* We improved security by removing microflow information from *metamodel.json*. Microflow information is now only available in the first session request (`get_session_data`) and only contains the microflows to which the user has access.
+* We improved the error message that was shown when trying to use a [Commit object](/refguide/committing-objects) activity on an external object. A suggestion is added to use the [Send external object](/refguide/send-external-object) activity instead, whenever the external entity allows for updates.
+* We added the possibility to specify the [BCrypt cost](/refguide/project-settings#bcrypt-cost) when configuring the hash algorithm via a custom setting. This will override the value from the model. To do this, use the value `BCRYPT:cost` for the setting `HashAlgorithm`, where `cost` can be a value from 10 to 30. If you only specify the value `BCRYPT`, a default `cost` value will be used.
+
```
[the remark this answers is not in the document]
Comment (Adam-Dupaski) [reply content/releasenotes/studio-pro/9.11.md:34]: @KevinVlaanderen Same note was already published for 8.18.16: https://github.com/mendix/docs/pull/4198/files

And will be published for 7.23.29: https://github.com/mendix/docs/pull/4199/files

Is this edit improving technical accuracy, or just making more minimal?

[the remark this answers is not in the document]
Comment (KevinVlaanderen) [reply content/releasenotes/studio-pro/9.11.md:34]: Ah ok, i didn't realize that. It was triggered by the fact that we did see these lines in the changelog file in appdev, but not in the GitHub MR. The assumption was that people added release notes after the freeze that then never made it into the GitHub MR

**Pull request merged**